app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to new_user_path
+    else
+      render new_user_path
```

Comment:
`render :new` is more appropriate here
You can read about their difference [here](https://medium.com/@kerenlerner/render-and-redirect-which-to-use-106ff653ee9a)